numpy/_core/src/umath/umathmodule.c

```diff
@@ -306,27 +306,33 @@ int initumath(PyObject *m)
      */
     int res = PyDict_GetItemStringRef(d, "logical_and", &s);
     if (res <= 0) {
+        Py_DECREF(s);
```

Comment:
s is NULL in this case and Py_DECREF(NULL) will crash.

Reply:
🙃 thanks again